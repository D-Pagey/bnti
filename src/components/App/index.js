import React from 'react';

export default function App() {
    return (
        <div className="App">
            <h1>Bantiiiii</h1>
            <p>API_GATEWAY = {process.env.REACT_APP_API_GATEWAY}</p>
        </div>
    );
}
