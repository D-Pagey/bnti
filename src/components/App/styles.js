import styled from 'styled-components';
import { colour, fontFamily } from '../../tokens';

export const Wrapper = styled.div`
    background-color: ${colour.black};
    font-family: ${fontFamily.body};
`;
