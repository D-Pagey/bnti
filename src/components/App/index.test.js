import React from 'react';
import { render } from 'react-testing-library';
import App from '.';

describe('App component', () => {
    it('should render', () => {
        const { container } = render(<App />);
        expect(container.firstChild).toMatchSnapshot();
    });
});
