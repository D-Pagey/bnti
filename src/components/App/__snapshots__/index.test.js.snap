// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render 1`] = `
.c5 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  list-style: none;
  margin: 0;
  padding: 0;
}

.c6 {
  margin-right: 2rem;
}

.c6:hover {
  color: white;
}

.c7 {
  color: inherit;
  font-size: 1.2rem;
  font-weight: 500;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c8 {
  background-color: black;
  border: 0;
  border-radius: 16px;
  color: white;
  cursor: pointer;
  font-size: 1.2rem;
  font-weight: 100;
  padding: 5px 15px;
}

.c1 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #00A7CB;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 1.3rem 1rem;
}

.c3 {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c2 {
  margin: 0;
}

.c4 {
  color: white;
}

.c9 {
  background-color: #353535;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 400px;
  padding: 1rem 0;
}

.c10 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin: 10px auto;
  width: 950px;
}

.c11 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 450px;
}

.c12 {
  color: #00A7CB;
  font-size: 36px;
  margin: 0 0 2rem 0;
  text-align: center;
}

.c13 {
  background: linear-gradient(180deg,#20fff2 0%,rgba(255,255,255,0) 100%),#00a7cb;
  border: 0;
  border-radius: 33px;
  cursor: pointer;
  font-size: 1.25rem;
  opacity: 0.9;
  padding: 1rem 1.35rem;
}

.c14 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 430px;
}

.c15 {
  height: auto;
  margin: 10px 20px;
  width: 250px;
}

.c16 {
  height: auto;
  margin: 10px 20px;
  width: 150px;
}

.c17 {
  background-color: #00A7CB;
  padding-top: 1.5rem;
  padding-right: 0rem;
  padding-bottom: 1.5rem;
  padding-left: 0rem;
  width: 100%;
}

.c18 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 2rem auto;
}

.c19 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-right: 1px solid #353535;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  max-width: 225px;
  padding-top: 1rem;
  padding-right: 1rem;
  padding-bottom: 2rem;
  padding-left: 1rem;
  width: 25%;
}

.c19:last-child {
  border: 0;
}

.c20 {
  height: auto;
  width: 100px;
}

.c23 {
  height: auto;
  width: 160px;
}

.c21 {
  font-size: 1.5rem;
  margin-top: 1rem;
  margin-right: 0rem;
  margin-bottom: 1rem;
  margin-left: 0rem;
  text-align: center;
}

.c22 {
  margin: 0rem;
  text-align: center;
}

.c0 {
  font-family: Quicksand,Tahoma,Verdana,sans-serif;
}

<div
  class="c0"
>
  <div
    class="c1"
  >
    <h2
      class="c2"
    >
      <a
        aria-current="page"
        class="c3 active"
        href="/"
      >
        <span
          class="c4"
        >
          BR
        </span>
        Battles
      </a>
    </h2>
    <ul
      class="c5"
    >
      <li
        class="c6"
      >
        <a
          class="c7"
          href="/about"
        >
          About
        </a>
      </li>
      <li
        class="c6"
      >
        <a
          class="c7"
          href="/tournaments"
        >
          Tournaments
        </a>
      </li>
      <li
        class="c6"
      >
        <a
          class="c7"
          href="/stats"
        >
          Stats
        </a>
      </li>
      <li
        class="c6"
      >
        <a
          class="c7"
          href="/signup"
        >
          <button
            class="c8"
            type="button"
          >
            Sign Up
          </button>
        </a>
      </li>
    </ul>
  </div>
  <div
    class="c9"
  >
    <div
      class="c10"
    >
      <div
        class="c11"
      >
        <h3
          class="c12"
        >
          Discover a new way to play your favourite games
        </h3>
        <a
          href="/tournaments"
        >
          <button
            class="c13"
            type="button"
          >
            Find games
          </button>
        </a>
      </div>
      <div
        class="c14"
      >
        <img
          alt="pubg"
          class="c15"
          src="logo-pubg.png"
          width="250px"
        />
        <img
          alt="apex"
          class="c16"
          src="logo-apex.png"
          width="150px"
        />
        <img
          alt="fortnite"
          class="c16"
          src="logo-fortnite.png"
          width="150px"
        />
      </div>
    </div>
  </div>
  <div
    class="c17"
  >
    <div
      class="c18"
    >
      <div
        class="c19"
      >
        <img
          alt="calendar"
          class="c20"
          src="calendar.png"
        />
        <div>
          <h3
            class="c21"
          >
            Step 1
          </h3>
          <p
            class="c22"
          >
            Find your tournament, you can filter by game and date.
          </p>
        </div>
      </div>
      <div
        class="c19"
      >
        <img
          alt="calendar"
          class="c20"
          src="chip.png"
        />
        <div>
          <h3
            class="c21"
          >
            Step 2
          </h3>
          <p
            class="c22"
          >
            Place your bounty to enter your tournament
          </p>
        </div>
      </div>
      <div
        class="c19"
      >
        <img
          alt="calendar"
          class="c23"
          src="loot.png"
          width="160px"
        />
        <div>
          <h3
            class="c21"
          >
            Step 3
          </h3>
          <p
            class="c22"
          >
            Kill players to earn their bounties, tracked by BRB.
          </p>
        </div>
      </div>
      <div
        class="c19"
      >
        <img
          alt="calendar"
          class="c20"
          src="3chips.png"
        />
        <div>
          <h3
            class="c21"
          >
            Step 4
          </h3>
          <p
            class="c22"
          >
            Cash in your bounties. Gloat (optional)
          </p>
        </div>
      </div>
    </div>
  </div>
  <h1>
    Bantiiiii
  </h1>
  <p>
    API_GATEWAY = 
  </p>
  <a
    href="/login"
  >
    Go to login
  </a>
</div>
`;
